### DISCUSSION TITLE
CONSULTA-GIT

### DISCUSSION THREAD

**Discussion by andromeda2075** [cpp-review-dune/introductory-review#14] [Q&A] [answered]
**¿Cuál era el comando específico de git para  varios archivos simultáneamente  de gitpod a mi repositorio y no hacerlo de uno por uno?**

**Comment by carlosal1015** [comment, marked as answer]
Hola @andromeda2075, una forma sería así con el subcomando `add` y la bandera `--all`
![image](https://user-images.githubusercontent.com/21283014/154827567-60e641d2-6e60-478c-86c4-8168f14fecd1.png)

**Reply by andromeda2075** [reply]
Ya lo hice, gracias!

**Reply by carlosal1015** [reply]
Ajá, la secuencia es `git add`, `git commit` y `git push`. En la práctica sería [así](https://man.archlinux.org/man/gittutorial.7).
